src/library_html5.js

```diff
@@ -988,13 +1008,14 @@ var LibraryHTML5 = {
   emscripten_set_orientationchange_callback_on_thread__proxy: 'sync',
   emscripten_set_orientationchange_callback_on_thread__deps: ['$registerOrientationChangeEventCallback'],
   emscripten_set_orientationchange_callback_on_thread: (userData, useCapture, callbackfunc, targetThread) => {
-    if (!screen || !screen['addEventListener']) return {{{ cDefs.EMSCRIPTEN_RESULT_NOT_SUPPORTED }}};
-    return registerOrientationChangeEventCallback(screen, userData, useCapture, callbackfunc, {{{ cDefs.EMSCRIPTEN_EVENT_ORIENTATIONCHANGE }}}, "orientationchange", targetThread);
+    if (!screen || !screen['orientation']) return {{{ cDefs.EMSCRIPTEN_RESULT_NOT_SUPPORTED }}};
+    return registerOrientationChangeEventCallback(screen.orientation, userData, useCapture, callbackfunc, {{{ cDefs.EMSCRIPTEN_EVENT_ORIENTATIONCHANGE }}}, "change", targetThread);
```

Comment:
This code line and the above mix use of `screen.orientation` and `screen['orientation']`. If Closure knows about `screen.orientation`, then using that would be preferred. If it hasn't been taught (I presume it should already have an externs file with that IDL in it), we'd use `screen['orientation']`.

So `screen['orientation']` -> `screen.orientation` above?

Or actually, I suppose these uses should both be `window.orientation` since `screen.orientation` is not referenced anywhere?

Reply:
`screen.orientation` works fine with Closure, as does `window.orientation` (surprisingly). What doesn't work are the prefixed versions, so I changed them changed to `window['webkitOrientation]`, etc.

`if (!screen || !screen['orientation'])` will fail though for the same reason `if (window)` fails (but any browser that doesn't have `window.screen` [won't support Emscripen at all](https://developer.mozilla.org/en-US/docs/Web/API/Window/screen#browser_compatibility)). Shouldn't this be `window['screen']`?